collector/src/main.rs

```diff
@@ -859,6 +876,11 @@ enum Commands {
         // toolchain name, and `PathBuf` doesn't work well for the latter.
         #[clap(long)]
         rustc2: Option<String>,
+
+        /// How many benchmarks should be profiled in parallel.
+        /// This flag is only supported for certain profilers.
```

Comment:
```suggestion
        /// This flag is only supported for certain profilers
```
Nit: no '.' at the end of help strings.

Reply:
`clap` removes the last `.` anyway :grin: But of course it's better for consistency :) Thanks.

Reply:
Oh, I didn't know that!